docs-src/socket-mode/index.rst

```diff
@@ -13,6 +13,21 @@ First off, let's start with enabling Socket Mode. Visit `the Slack App configura
 
 * Go to **Settings** > **Basic Information**, then add a new **App-Level Token** with the `connections:write` scope
 * Go to **Settings** > **Socket Mode**, then turn on **Enable Socket Mode**
+* Go to **Features** > **App Home**, look under **Show Tabs** > **Messages Tab** then turn on **Allow users to send Slash commands and messages from the messages tab**
+* Go to **Features** > **Event Subscriptions**, then turn on **Enable Events**
+
```

Comment:
```suggestion
```

Reply:
I think this and the change on L:20 would make the generated page look like this 
![image](https://user-images.githubusercontent.com/14988311/143802368-5645045f-2b25-4f14-9d25-aa2a8212ac99.png)

with the current file `./scripts/docs.sh` generates the following for me 
![image](https://user-images.githubusercontent.com/14988311/143802449-f4669cdf-c5b1-409c-9367-7021cea1d4a8.png)

if it still looks good on your end with the proposed changes I'll commit them, but I wanted to bring it up since it looks like the formatting might be wonky to me

Reply:
@ruberVulpes Thanks for clarifying this! Your changes look reasonable here.